src/component/autocomplete.js

```diff
@@ -150,8 +152,8 @@ export class AutoCompleteCustomElement {
   labelWithMatches(result) {
     let label = this.label(result);
 
-    if (!label.replace) {
-      return '';
+    if (typeof label !== 'string') {
+      return ' ';
```

Comment:
I will document that the reason for this is because it avoids the element not getting the proper height.